### ISSUE TITLE
Genome (-g) files are ignored when BAM input is provided

### ISSUE THREAD

**Issue by VikArz02** [iMetOsaka/UNAGI#6]
Hi,
I try to run the tool you made but have this error:

`[M::main] Real time: 329.979 sec; CPU: 799.731 sec; Peak RSS: 18.511 GB
[2021/03/23 - 12:15:54] Sorting the new mapped reads
[2021/03/23 - 12:16:38] Generating the genome coverage for each position

*****
*****WARNING: Genome (-g) files are ignored when BAM input is provided. 
*****
`
Can you help me?

**Comment by JungNicolas** [comment]
The warning should not be showing anymore with the latest version.
Thank you for reporting this problem !

**Comment by VikArz02** [comment]
Yup, this problem doesn't appear, but the program just killed in the same place.

**Comment by JungNicolas** [comment]
Hi ! 
Oh, so the program just exits with no other message ? I can't really tell what's wrong from just this, but let's try to find the reason.

First thing, just in case you were killing it yourself: the program normally takes some time finishing the "Generating the genome coverage for each position" step, so please don't assume that it is hanging because of that, it will eventually finish. 

In case this was not your problem, could you please do the following to help me pinpoint the culprit: 
- run the program with the "-V" or "--verbose" option on and show me the last few lines of console output.  
- try to run the program with the test files now included in the package (you can find them in UNAGI/tests) and tell me if it crashes in the same way.

Thanks for your patience, I hope I can manage to identify and fix the issue quickly.
